### ISSUE TITLE
How to train a multilingual model, is there a script for it?

### ISSUE THREAD

**Issue by zds-potato** [wenet-e2e/wenet#1656]
I see that the w2v-conformer pre-trained model is trained using a multilingual dataset. Currently I have not found a relevant multilingual training solution or script.

Some of the problems encountered so far are how to choose the text modeling unit, is it BPE or char or something else?

**Comment by Emiyassstar** [comment]
w2v-conformer don't use any text information to calculate pretrain loss . But in order not to change the wenet training pipeline,you can fill in any text unit just like 'A' for  multilingual wavs.
For multilingual training, You can merge all wavs into one dataset and balance data refer to Facebook's XLSR model.
For wav2vec training example ,you can see  https://github.com/wenet-e2e/wenet/pull/1003
